src/tools/rest-frontend/resources/structure.json

```diff
@@ -0,0 +1,1896 @@
+[
+    {
```

Comment:
Yes, I know the standard does not allow it. I cannot remember if the question was about this specific file. Then please add a "info" field (simply to be ignored)? It should contain explanations e.g. why not making it pretty or that there is actually no end marker. The structure.json.in will be a quite important file to maintain, not having comments there would be a drawback.